perl.h

```diff
@@ -543,11 +543,6 @@ __typeof__ and nothing else.
 #  endif
 #endif
 
-#if defined(_MSC_VER) && _MSC_VER < 1400
-/* XXX older MSVC versions have a smallish macro buffer */
-#  define PERL_SMALL_MACRO_BUFFER
-#endif
-
```

Comment:
It should be possible to eliminate all PERL_SMALL_MACRO_BUFFER references, since the macro was added to support older MSVC and it isn't defined anywhere else.

@khw might have an opinion here.